Vipul Kumar/if else/06_youngest.cpp

```diff
@@ -0,0 +1,24 @@
+#include<iostream>
+using namespace std;
+
+int main()
+{
+int a,b,c;
+cin>>a>>b>>c;
+if (a>b&&a>c)
+{
+    cout<<a<<"is oldest";
+}
+else if (b>a&&b>c)
```

Comment:
it can be simplify, as in first if condition is not true that means a is not our and, then we can simply check other two like, else if(b > c) .... else {]